react/features/base/config/reducer.js

```diff
@@ -172,6 +173,27 @@ function _setConfig(state, { config }) {
     return equals(state, newState) ? state : newState;
 }
 
+/**
+ * Processes the conferenceInfo object against the defaults.
+ *
+ * @param {Object} config - The old config.
+ * @returns {Object} The processed conferenceInfo object.
+ */
+function _getConferenceInfo(config) {
```

Comment:
duplicate of https://github.com/jitsi/jitsi-meet/pull/9638/files#diff-390047759702dc636ad45f0c08e411558c4c9a6bab1c812d0c9fd299fb479c93R11

Reply:
hmm one is a selector the other receives a config object as param, but I agree that some duplication exists. The thing is, those two conceptually are different, one is just as a helper for backwards compat, while the other is a selector used in to retrieve the current values for those configs, so I would not mix them. by exporting one to be used by the other